install/environment.cli.yml

```diff
@@ -0,0 +1,6 @@
+name: zarp
+channels:
+  - conda-forge
```

Comment:
The package is on Bioconda, so I don't think we need `conda-forge` here? Or am I missing something?

Reply:
removing the conda-forge gives back this error in particular: 

```
Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.17=0
  - feature:|@/linux-64::__glibc==2.17=0
```